spot_rl_experiments/configs/config.yaml

```diff
@@ -3,9 +3,13 @@ WEIGHTS:
   # GAZE: "weights/gaze_normal_32_seed300_1649708927_ckpt.49.pth"
   # PLACE: "weights/speed_sweep_seed2_speed0.174533_1648669786.ckpt.94.pth"
   # MIXER: "weights/sweep_17_SD_200_1651156250_ckpt.12.pth"
-# new
+
   # NAV: "weights/final_paper/nav_CUTOUT_WT_True_SD_300_ckpt.90.pth"
```

Comment:
Can we clean up these commented out lines?

Reply:
done @zephirefaith